hotel_booking.py

```diff
@@ -0,0 +1,210 @@
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+import numpy as np
+
+data_filepath = "hotel_bookings.csv"
+df = pd.read_csv(data_filepath)
+print("\ndf.columns \n=",df.columns)
```

Comment:
No reason to have \n all over the place. 

Also, fun to know. A trend in many languages is to use "Formatted string literals". In Python it would look like:
`print(f'df.columns={df.columns}')`